### PR TITLE
Bug 1309597 - Add periodic page refresh for dashboard, cluster and job detail pages

### PR DESCRIPTION
**Pull request by jezdez**
This change is [Reviewable](https://reviewable.io/reviews/mozilla/telemetry-analysis-service/46)

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by jezdez** [comment]
me writing JS: w8vnhzngir3mu

**Comment by jezdez** [comment]
screen recording 2016-10-18 at 06 24 pm

File: atmo/static/js/refresher.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+              var countdown_label = 'ing now';
+            } else {
+              var countdown_label = ' in ' + countdown + 's';
+            }
+            $(refresh_timer).text(countdown_label);
+          }
+          timeout_id = window.setTimeout(updateTimeout, 1000);
+        };
+        // show the refresher
+        console.log("Refresher init.");
```
Comment (maurodoglio) [inline atmo/static/js/refresher.js]: please remove debug statement

Comment (jezdez) [inline atmo/static/js/refresher.js]: 👍

**Pull request merged**